packages/blade/src/components/Card/__tests__/Card.native.test.tsx

```diff
@@ -171,4 +171,55 @@ describe('<Card />', () => {
       '[Blade Card]: CardFooterTrailing cannot be used outside of Card component',
     );
   });
+
+  it('should restrict childrens & only allow CardHeader, CardBody & CardFooter in Card', () => {
+    expect(() =>
+      renderWithTheme(
+        <Card>
+          <div>some random div</div>
+          <CardHeader />
+          <CardBody>Plain Card</CardBody>
+          <CardFooter />
+        </Card>,
+      ),
+    ).toThrow(
+      '[Blade Card]: Only one of `CardHeader, CardBody, CardFooter` component is accepted in Card children',
+    );
+  });
+
+  it('should restrict childrens in CardHeader', () => {
+    expect(() =>
+      renderWithTheme(
+        <Card>
+          <CardHeader>
+            <CardHeaderLeading title="" />
+            <div>some random children</div>
```

Comment:
there's never any div in Native so it will regardless throw error. can use Box instead for both web and native

Reply:
Right!